### ISSUE TITLE
[Feature Request] Keybind to toggle step assistance

### ISSUE THREAD

**Issue by egirlcatnip** [SecretOnline/accessible-step#9]

**Comment by SecretOnline** [comment]
Hi, thanks for the feature request! Adding a keybind isn't something I had considered for this mod originally, so I'm interested in what situations you'd likely be using it while playing the game. Could you give a bit more detail as to why it'd be useful?

**Comment by egirlcatnip** [comment]
Yes of course. The main reason would probably be to have a handy shortcut directly in game so that when I join a server which blocks this because of anticheat (eg. hypixel), i don't have to manually go change the setting. 
I jump in and out of singleplayer and mltiplayer and this would come in very handy. 

Additionally, if you consider this, I suggest 2 keybinds:
1. No step - Step
2. Cycle: No step - Step - Vanilla Auto Jump

**Comment by SecretOnline** [comment]
Thanks for the detail. Since it seems like you're using different settings between worlds, would it be more useful to be able to configure it once for each world/server, or do you find yourself turning it on and off during regular gameplay (i.e. while playing on a single world, for which a keybind would be the way to go)?

**Comment by egirlcatnip** [comment]
I do turn it on and off during regular gameplay as well. I don't mind leaving the keybind unasigned by default.

**Comment by SecretOnline** [comment]
Awesome, good to get the clarification.

I don't know when I'll get the time to look into this properly, but I think this is something that could be added. I'm saying this because it might be a while before I add it, but I want you to know that I won't forget. I might also create a separate issue for per-world configuration as well, but that's likely much further down the road.

When I do get time to work on it, I'll be sure to update you here.

**Comment by egirlcatnip** [comment]
okay, thank you a lot! ❤️

**Comment by SecretOnline** [comment]
Hello, I had some time (and importantly motivation) over the holidays, and I've added a keybind to the mod to cycle between step assistance modes. It's not bound by default, so you'll need to choose a key yourself.

I've also added per-server configuration, so if you go into the mod's settings page while connected to a server you can click the "Custom Config for World" button to create a separate profile for that server.
